modules/serverpod_auth/serverpod_auth_client/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ issue_tracker: https://github.com/serverpod/serverpod/issues
 environment:
   sdk: '>=3.1.0 <4.0.0'
 dependencies:
-  lints: ^3.0.0
+  lints: '>=3.0.0 <5.0.0'
```

Comment:
Suggestion: move to dev_dependencies

Reply:
Couldn't move these out since they are actually used by the project.